extensions/amp-carousel/0.1/base-carousel.js

```diff
@@ -81,6 +87,12 @@ export class BaseCarousel extends AMP.BaseElement {
     this.nextButton_.classList.add('amp-carousel-button-next');
     this.nextButton_.setAttribute('role', 'button');
     this.nextButton_.setAttribute('aria-label', 'Next item in carousel');
+    this.nextButton_.setAttribute('tabindex', 0);
+    this.nextButton_.onkeydown = event => {
+      if (event.keyCode == 13 /* enter */ || event.keyCode == 32 /* space */) {
+        this.interactionNext();
```

Comment:
not really needed here (but won't hurt) since there is no other handler registered that can prevent default.